CreatubblesAPIClient/Sources/Model/Notification.swift

```diff
@@ -91,4 +91,13 @@ open class Notification: NSObject, Identifiable
         bubbleRelationship = MappingUtils.relationshipFromMapper(mapper.bubbleRelationship)
         bubble = MappingUtils.objectFromMapper(dataMapper, relationship: bubbleRelationship, type: Bubble.self)
     }
+    
+    var isValid: Bool
```

Comment:
couldn't this just be:
```
(creation != nil && !(creationEntities?.isEmpty) || 
user != nil && !(userEntities?.isEmpty) || 
gallery != nil && !(galleryEntities?.isEmpty))
```
Removes all force unwraps (unnecessary here), and achieves the same thing in less code, right?